### ISSUE TITLE
[WhyPaused] Style the UI

### ISSUE THREAD

**Issue by jasonLaster** [devtools-html/debugger.html#1604]
There are a couple things that are currently off with the UI:

* the height is should be consistent with headers
* there should be a bottom bar
* the color is arbitrary, we should choose standard colors that look good in dark mode
* we should check to see that it looks good in vertical mode
* we should make sure it looks good while scrolling

```patch
diff --git a/src/components/WhyPaused.js b/src/components/WhyPaused.js
index 6b4f90d..5d2910e 100644
--- a/src/components/WhyPaused.js
+++ b/src/components/WhyPaused.js
@@ -21,6 +21,7 @@ const WhyPaused = React.createClass({
     const { pauseInfo } = this.props;
     const reason = getPauseReason(pauseInfo);

+    // => here
     return reason ?
       dom.div({ className: "pane why-paused" }, L10N.getStr(reason))
       : null;
```

* scrolling

![](http://g.recordit.co/3hBYHzO5Ag.gif)

* vertical mode

![screen shot 2017-01-03 at 5 16 36 pm](https://cloud.githubusercontent.com/assets/254562/21625174/6fa8a528-d1d8-11e6-9ec8-122ba3aa5fde.png)

**Comment by krishnagopinath** [comment]
I'd like to work on this issue, @jasonLater, if it's not already taken by someone else :)

This would be my first ever contribution to OS ever,  so I'm pretty sure I'd make a lot of whoopsies.

**Comment by jasonLaster** [comment]
Hi @krishnagopinath. It's yours, feel free to reach out with any questions

**Comment by krishnagopinath** [comment]
So, I've got a couple of suggestions for this (thanks to the chat I had with @jasonLaster ), other than reducing the font size, and the height, and adding borders :

* The red dot - Here's a little preview as to how it'll look.. of course, this looks rough and ugly but once I've done some CSS work on it, it'll look much better! (Look for the red dot next to the "Paused on breakpoint" message

![why-paused-badge](https://cloud.githubusercontent.com/assets/4935809/21704211/c42b2da6-d385-11e6-9bb4-6a75b16d187d.png)

If I were to consider adding the small red dot before the text in the `why-paused` div, I'd have to use an SVG, from the looks of it. [Can I use an SVG from anywhere on the web?](https://commons.wikimedia.org/wiki/File:Location_dot_red.svg) 

I see that other SVGs in the `assets/images` folder have an MPL as the first line, hence the question. 

* Set the message background to whatever the `--breakpoint-active-color` is. This would make sure that the user will know that this is paused, and using the same color will unify the message intent with the screen visual. I haven't looked at the entire code base, but I'm assuming the `--breakpoint-active-color` is different for the "dark" mode, which will solve the problem of visibility in the dark mode. 

![whypaused-break](https://cloud.githubusercontent.com/assets/4935809/21704353/d0737892-d386-11e6-9e22-3206f5b750d7.png)

Thoughts?

**Comment by clarkbw** [comment]
I'd leave out the red dot.  Our goal isn't to _draw attention_ to this space as much as it is to give the user information if they are unsure why the debugger is paused. 

I like matching the breakpoint color, that makes sense to me, perhaps even dropping the opacity on that color by a bit so it fades out more than the actual breakpoint. 

And cleaning up the padding etc is def a good thing.

With this cleaned up we could start looking at #1611 next!

**Comment by jasonLaster** [comment]
@nchevobbe thoughts here?

**Comment by nchevobbe** [comment]
Tweaked things a little :)

![peek 06-01-2017 17-05](https://cloud.githubusercontent.com/assets/578107/21724151/8142d846-d433-11e6-9f89-39ec5e883c50.gif)

Like @krishnagopinath  I think it makes sense to have a similar hue to the one we're using for the paused-line background.
I have maybe an issue with using green to indicate something is paused, because in my mind green would mean "running" (like in traffic lights), but it's a bit out of scope here.

I think centering the label makes sense so it doesn't feel like the other label on the break point and call stack panels. I also reduced the font-size to be the same as the panel titles, i think it was a bit to prominent with the higher font-size.

I added some darker border in the same shade than the background so it feels a little more "on top" (don't know how to express it well) of the scrolled elements. The border could be shorter, or of another color, or both. I can test other combination to see what we like best.

Maybe adding a little paused ("II") icon in front of it would help, but I'm not sure since it could clash with the one in the upper toolbar.

Tell me what you think about it
